nusight2/src/client/components/storybook/fullscreen.tsx

```diff
@@ -1,16 +1,5 @@
 import React from 'react'
 
-import style from './fullscreen.css'
-
 export function fullscreen(story: () => JSX.Element) {
-  return (
-    <div className={style.fullscreen}>
-      {
-        // This is an inline <style> and not in `fullscreen.css` as that would make it apply globally
-        // to all stories, due to the way we build and extract the styles from CSS files.
-      }
-      <style>{'body { margin: 0; padding: 0; }'}</style>
-      {story()}
-    </div>
-  )
+  return <div style={{ width: '100vw', height: '100vh' }}>{story()}</div>
```

Comment:
Is there an example of a story which doesn't work without this decorator? Can we remove the decorator altogether?

Reply:
Removing the `fullscreen` decorator breaks these stories (no image):

- vision objects
- vision grid_layout (both static and animated)

I also initially had trouble with the animated three stories (nugus and darwin model, three animated scene) - it would render into a small rectangle in the middle of the screen. Now it flashes like that for a moment and then goes fullscreen.

Reply:
Gonna merge now. Happy to follow up with improvements.